docs/source/tutorials/faq.ipynb

```diff
@@ -550,6 +550,55 @@
     "print(cl.clf)  # will be RandomizedSearchCV(estimator=GradientBoostingClassifier(),...)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "d228decd",
+   "metadata": {},
+   "source": [
+    "### Why does regression.learn.CleanLearning take so long?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "de5c984b",
+   "metadata": {},
+   "source": [
+    "To more effective identify and score datapoints in a regression dataset, the methods in [regression.learn.CleanLearning](../../cleanlab/regression/learn.html#cleanlab.regression.learn.CleanLearning) estimate each datapoint's aleatoric uncertainty (by fitting a second copy of the model to predict the residuals’ magnitudes), as well as its epistemic uncertainty (by fitting multiple copies of the model with bootstrap resampling). The residual of each datapoint is then adjusted by these estimated to provide a robust quality score that accounts for the model's prediction errors. \n",
+    "\n",
+    "These estimated are expected to produce better results but will require longer runtimes. If you would like to speed up the runtime of these methods, here are a few options:\n",
+    "\n",
+    "- Reducing the number of bootstrap sample rounds by reducing passing in a lower number of sampling rounds in the `n_boot` argument (default value is 5), or setting to 0 to skip the epistemic uncertainty estimation entirely.\n",
+    "\n",
+    "- Setting `include_aleatoric_uncertainty=False` to skip the aleatoric uncertainty estimation.\n",
+    "\n",
+    "- Having less elements in the `coarse_search_range` of [regression.learn.CleanLearning.find_label_issues](../cleanlab/regression/learn.html#cleanlab.regression.learn.CleanLearning.find_label_issues). This is the coarse search range to find the value of `k`, which estimates the fraction of data that have label issues. More values represent a more thorough search, hence we can reduce the runtime of this method by reducing this search range.\n",
+    "\n",
+    "- Reducing the `fine_search_size` in [regression.learn.CleanLearning.find_label_issues](../cleanlab/regression/learn.html#cleanlab.regression.learn.CleanLearning.find_label_issues). This is the size of the fine-grained search grid to find the value of `k`, which estimates the fraction of data that have label issues. A higher number represent a more thorough search, hence we can reduce the runtime of this method by using a smaller search size.\n",
+    "\n",
+    "Below is sample code on how to pass in these arguments."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "623df36d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from cleanlab.regression.learn import CleanLearning\n",
+    "\n",
+    "X = np.random.random(size=(200, 3))\n",
```

Comment:
```suggestion
    "X = np.random.random(size=(30, 3))\n",
```

Reply:
will 30 work, or do we need bigger value like 200?

Reply:
prefer these run instantly

Reply:
Yup 30 works, will change this